x/bridge/types/constants.go

```diff
@@ -6,4 +6,6 @@ const (
 	DefaultBitcoinConfirmations = 6
 	DefaultBitcoinExponent      = 10 // TODO: decide
 	DefaultVotesNeeded          = 1
+
+	DefaultOsmosisChainName = "osmosis"
```

Comment:
Should we be referring to chains (at least cosmos chains) by their chainID (in this case osmosis-1)?